docs/user/api/cross-site-requests.rst

```diff
@@ -0,0 +1,30 @@
+Cross-site requests
+===================
+
+Cross site requests are allowed for the following endpoints:
+
+- :ref:`/api/v2/footer_html/ <api/v2:Undocumented resources and endpoints>`
+- :ref:`/api/v2/search/ <server-side-search/api:API V2 (deprecated)>`
+- :ref:`/api/v2/embed/ <api/v2:Embed>`
+- :ref:`/api/v2/sustainability/ <api/v2:Undocumented resources and endpoints>`
+- :ref:`/api/v3/embed/ <api/v3:Embed>`
+
+Except for the sustainability API, all of the above endpoints
+don't allow you to pass credentials in cross-site requests.
+This means you can make use of those APIs to get information from public versions only.
```

Comment:
```suggestion
In other words, these API endpoints *only* expose public information.

On a technical level, this is achieved by implementing the CORS_ standard which is supported by all major browsers. We implement it such that *origins*, *protocols* and *credentials* strictly match the intention of the API endpoint.

.. _CORS: https://en.wikipedia.org/wiki/Cross-origin_resource_sharing
```

Reply:
Added this but without mentioning the part about origins, we allow all origins :D and not sure about the part of protocols, maybe you meant the http verbs?

Reply:
The new text sounds great, thanks for implementing it :+1: